### DISCUSSION TITLE
Plugin developer documentation

### DISCUSSION THREAD

**Discussion by JudoCoder** [lapce/lapce#1537] [Q&A]
Is there any plugin developer documentation?

**Comment by panekj** [comment]
Currently no

**Reply by coffeeispower** [reply]
I think we don't even need to think why

**Reply by JudoCoder** [reply]
Sorry … what does that mean?

**Reply by maxpetts** [reply]
The Plugin API is subject to change @JudoCoder. However, there are [templates](https://github.com/lapce-community/lapce-volt-template) to get you started. Documentation is in the works but a guide, working as of 21/12/22, is available [here](https://github.com/lapce-community/lapce-volt-template/blob/3134ed6fc279af29f178b00d5965cf67cd9d8345/README.md)

**Reply by JudoCoder** [reply]
Thank you, maxpetts.
